app/Models/User.php

```diff
@@ -133,6 +136,14 @@ public function permissions(): HasManyDeep
         return $this->hasManyDeepFromRelations($this->roles(), (new Role())->permissions());
     }
 
+    public function caseItems()
+    {
+        return $this->hasMany(
+            related: CaseItem::class,
+            foreignKey: 'assigned_to_id',
+        );
+    }
```

Comment:
Is it also possible that this needs to be a many-to-many relationship? Or is there only ever one case owner at a time? Do we need a better way other than auditing to keep the historical trail that comes from re-assignment that we can reference?

```suggestion
    public function caseItems(): HasMany
    {
        return $this->hasMany(
            related: CaseItem::class,
            foreignKey: 'assigned_to_id',
        );
    }
```